packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2417,6 +2484,42 @@ class RenderFittedBox extends RenderProxyBox {
 
   // TODO(ianh): The intrinsic dimensions of this box are wrong.
```

Comment:
Can this TODO be removed?

Reply:
The other intrinsics in this file are still incorrect.